grovedb/src/replication.rs

```diff
@@ -138,7 +138,7 @@ impl<'db> Restorer<'db> {
                 Merk::open_base(grove_db.db.get_storage_context(empty()).unwrap())
                     .unwrap()
                     .map_err(|e| RestorerError(e.to_string()))?,
-                root_key,
+                None,
```

Comment:
Why?

Reply:
The root tree is not bounded by an Element::Tree node as such it doesn't need a combining value.